events/janus_wsevh.c

```diff
@@ -575,60 +571,53 @@ json_t *janus_wsevh_handle_request(json_t *request) {
 		}
 }
 
-/* Thread to implement the WebSockets loop */
-static void *janus_wsevh_thread(void *data) {
-	JANUS_LOG(LOG_VERB, "Joining WebSocketsEventHandler client thread\n");
-	while(g_atomic_int_get(&initialized) && !g_atomic_int_get(&stopping)) {
-		/* Loop until we have to stop */
-		if(!reconnect) {
-			lws_service(context, 50);
-		} else {
-			/* We should reconnect, get rid of the previous context */
-			if(reconnect_retries > 0) {
-				/* Wait a few seconds before retrying */
-				gint64 now = janus_get_monotonic_time();
-				if((now-disconnected) < (gint64)reconnect_retries*G_USEC_PER_SEC) {
-					/* Try again later */
-					g_usleep(100000);
-					continue;
+#if (LWS_LIBRARY_VERSION_MAJOR >= 3 && LWS_LIBRARY_VERSION_MINOR > 2) || (LWS_LIBRARY_VERSION_MAJOR > 3)
+	/*
+	* Websocket thread loop for websocket library newer thant 3.2
+	* The reconnect is handled in a dedicated wsl scheduler janus_wsevh_schedule_connect_attempt
```

Comment:
I guess _wsl_ is a typo ?

Reply:
yeah :) and copy pasted, fixing it everywhere